FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -47,23 +47,31 @@ TextField {
     -fx-skin: "com.pixelduke.control.skin.FXTextFieldSkin";
 }
 
-TextField {
-    -right-button-visible: true;
```

Comment:
We need to have the -right-button-visible css property here. This is also for documentation reasons, so that users know this css property exists and that the default value is true.

Reply:
This may have been an oversight, it can be added back

Reply:
Added back in 48cb59e